paymentsheet/api/paymentsheet.api

```diff
@@ -290,6 +291,7 @@ public final class com/stripe/android/paymentsheet/PaymentSheet$Configuration :
 	public final fun getAllowsDelayedPaymentMethods ()Z
 	public final fun getAllowsPaymentMethodsRequiringShippingAddress ()Z
 	public final fun getAppearance ()Lcom/stripe/android/paymentsheet/PaymentSheet$Appearance;
+	public final fun getBillingDetailsCollectionConfiguration ()Lcom/stripe/android/ui/core/BillingDetailsCollectionConfiguration;
```

Comment:
It's weird that we expose these publicly. Might as well keep being consistent. But it's another thing we should clean up on the next major version.